include/OpenColorIO/OpenColorTypes.h

```diff
@@ -263,6 +261,25 @@ OCIO_NAMESPACE_ENTER
         INTERP_BEST = 255       //! the 'best' suitable interpolation type
     };
     
+    //!cpp:type::
+    //
+    // Specify the method to use when inverting a Lut1D or Lut3D.  The BEST
+    // method is slower, and only available on the CPU, but it calculates an
+    // exact inverse.  The exact inverse is based on the use of LINEAR forward
+    // interpolation for Lut1D and TETRAHEDRAL forward interpolation for Lut3D.
+    // The FAST method bakes the inverse into another forward LUT (using the
+    // exact method).  For Lut1D, a half-domain LUT is used and so this is
+    // quite accurate even for scene-linear values, but for Lut3D the baked
+    // version is more of an approximation.  The DEFAULT is the fast method
+    // since it is the only one available on both CPU and GPU.
+    //
+    enum LutInversionQuality
+    {
+        LUT_INVERSION_BEST = 0,
+        LUT_INVERSION_FAST,
+        LUT_INVERSION_DEFAULT
+    };
```

Comment:
To be consistent with Interpolation, I feel like this should be:

LUT_INVERSION_EXACT = 0,
LUT_INVERSION_FAST,

LUT_INVERSION_DEFAULT = 254,
LUT_INVERSION_BEST = 255

BEST would be an alias to the "best" suitable inversion quality (EXACT), and DEFAULT an alias to the default inversion quality (FAST). Neither should describe an inversion quality themselves.